ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/clientlibs/site/js/checkboxgroupview.js

```diff
@@ -71,6 +71,19 @@
             return this.element.querySelector(CheckBoxGroup.selectors.tooltipDiv);
         }
 
+        syncWidget() {
+            let widgetElementList = this.getWidget();
+            if(widgetElementList) {
+                widgetElementList.forEach((widgetElement) => {
+                    let idToReplace=widgetElement.id.split('_')[0];
+                    widgetElement.id=widgetElement.id.replace(idToReplace,this.getId());
+                    if(widgetElement.parentElement.tagName.toLowerCase()==='label') {
```

Comment:
we should find the label element and then set its attribute. this code is being rigid about present of label in parent element and that too as a label tag.

Reply:
Hi @nit23uec , we have already handled the common label case in FormFIeldBase.
https://github.com/adobe/aem-core-forms-components/blob/master/ui.frontend/src/view/FormField.js#L84

 in case of checkboxgroup this is not the component label that the view 's getLabel returns, this is specific label to the items  widget in the checkboxgroup, that's why this is handled separately only inside checkboxgroup
 
 cc:- @sakshi-arora1